src/Storage/ABS/SnowflakeExportAdapter.php

```diff
@@ -39,18 +39,6 @@ public function runCopyCommand(
         Storage\DestinationInterface $destination,
         ExportOptions $exportOptions
     ): void {
-        if (!$source instanceof Storage\SqlSourceInterface) {
```

Comment:
Jde to pryč protože se předtím kontroluje v `isSupported`?

Reply:
jj řeší to isSupported